packages/svelte-undp-design/src/lib/Breadcrumbs/Breadcrumbs.svelte

```diff
@@ -30,7 +30,7 @@
 									<img src={breadcrumb.icon} alt="{breadcrumb.name}_image" />
 								{/if}
 							</span>
-							<span>{breadcrumb.name}</span>
+							<span style="font-size:1rem">{breadcrumb.name}</span>
```

Comment:
Can you make this setting as a parameter rather than hard coding?

like `<Breadcrumbs fontSize={'small' | 'medium' | 'normall'} />`

Reply:
https://bulma.io/documentation/helpers/typography-helpers/#size

you can also check bulma/s typography